components/03-modules/additional-content/AdditionalContent.stories.js

```diff
@@ -0,0 +1,25 @@
+import { storiesOf } from '@storybook/vue'
+
+import App from '../../01-globals/app/App.vue'
+import AlpacaAdditionalContent from './AdditionalContent.vue'
+
+import picture from './mocks/picture.json'
+
+storiesOf('Modules/Additional Content', module).add('Default', () => ({
+  components: { App, AlpacaAdditionalContent },
+  data () {
+    return {
+      picture
+    }
+  },
+  template: `
+    <app>
+      <alpaca-additional-content
+        :picture="picture"
+        pictureAlt="Sample alt text"
+        firstColumnText="First column"
+        secondColumnText="Second column"
```

Comment:
https://vuejs.github.io/eslint-plugin-vue/rules/attribute-hyphenation.html#rule-details for all above props

Reply:
Done